src/bot/commands/random/trivia.ts

```diff
@@ -141,108 +153,207 @@ export class Trivia extends Command {
         let parsed = await parseResponse(openTDB + questionAmount + categoryURL + difficultyURL + typeURL) as TriviaResponse;
         let count = 0;
         //do {
-            
-            //Fix any encoding errors using entities.decode()
-            //Normalize the names for the parsed TriviaResponse data 
-            let question = entities.decode(parsed.results[count].question);
-            let incorrect_answers: string[] = [];
-            let correct_answer = entities.decode(parsed.results[count].correct_answer);
-            let answers = [correct_answer];
-            _.each(parsed.results[count].incorrect_answers, s => incorrect_answers.push(entities.decode(s)));
-            incorrect_answers.forEach(element => { answers.push(element); });
-
-            // console.log(`Question: ${question}`);
-            // console.log(`dQuestion: ${entities.decode(question)}`);
-            // console.log(`CorrectAnswer: ${correct_answer}`);
-            // console.log(`IncorrectAnswers: ${incorrect_answers}`);
-            // console.log(`Category: ${parsed.results[count].category}`);
-
-            //Randomize answers before displaying
-            if (answers.length > 2) {
-                for (let i = answers.length - 1; i > 0; i--) {
-                    let e = Math.floor(Math.random() * (i + 1));
-                    let temp = answers[i];
-                    answers[i] = answers[e];
-                    answers[e] = temp;
-                }
+
+        //Fix any encoding errors using entities.decode()
+        //Normalize the names for the parsed TriviaResponse data 
+        let question = entities.decode(parsed.results[count].question);
+        let incorrect_answers: string[] = [];
+        let correct_answer = entities.decode(parsed.results[count].correct_answer);
+        let answers = [correct_answer];
+        _.each(parsed.results[count].incorrect_answers, s => incorrect_answers.push(entities.decode(s)));
+        incorrect_answers.forEach(element => { answers.push(element); });
+
+
+        // console.log(`Question: ${question}`);
+        // console.log(`dQuestion: ${entities.decode(question)}`);
+        // console.log(`CorrectAnswer: ${correct_answer}`);
+        // console.log(`IncorrectAnswers: ${incorrect_answers}`);
+        // console.log(`Category: ${parsed.results[count].category}`);
+
+        //Randomize answers before displaying
+        if (answers.length > 2) {
+            for (let i = answers.length - 1; i > 0; i--) {
+                let e = Math.floor(Math.random() * (i + 1));
```

Comment:
In the future, you can use the [`_.shuffle`](https://lodash.com/docs/4.17.11#shuffle) method in lodash to randomize an array.

Reply:
yeah realized that after Nathan brought it up